docs/faq/index.rst

```diff
@@ -86,5 +42,17 @@ Your message will be forwarded to the ClusterHQ security team (a small group of
 Due to the sensitive nature of security issues, we ask you not to send a message to one of the public mailing lists.
 ClusterHQ has a policy for :ref:`reporting-security-issues` designed to minimize any damage that could be inflicted through public knowledge of a defect while it is still outstanding.
 
-.. _good write up of the ZFS and Linux license issues: http://zfsonlinux.org/faq.html#WhatAboutTheLicensingIssue
+ZFS
+~~~
+
+.. note:: The Flocker ZFS backend is Experimental, and should not be used in a production environment.
+
```

Comment:
Grammar :) "The Flocker ZFS backend is experimental and should not be used in a production environment."
Also, change to a warning I think.

Like @wallrj I'd be inclined to not have a ZoL section in the FAQ at all, but if it's staying, make the small change above.